gen/gen.go

```diff
@@ -7,18 +7,20 @@ import (
 	"fmt"
 	"go/format"
 	"io/ioutil"
+	"log"
+	"net/http"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"runtime"
 	"text/template"
+	"time"
 )
 
-// dirMetadata is the location of the x-common repository
-// on the filesystem.
-// We're making the assumption that the x-common repository
-// has been cloned to the same parent directory as the xgo
-// repository. E.g.
+// dirMetadata is the location of the x-common repository on the filesystem.
```

Comment:
It would be good if these were in a separate commit since they have nothing to do with collecting canonical data remotely.